app/views/internal/configs/show.html.erb

```diff
@@ -689,6 +688,26 @@
           </div>
         </div>
 
+        <div class="card mt-3">
+          <%= render partial: "card_header",
+                     locals: {
+                       header: "User Experience",
+                       state: "collapse",
+                       target: "uxBodyContainer",
+                       expanded: "false"
+                     } %>
+          <div id="uxBodyContainer" class="card-body collapse hide" aria-labelledby="uxBodyContainer">
+            <div class="form-group">
+              <%= f.label :feed_style %>
+              <%= f.text_field :feed_style,
```

Comment:
Since there is a limited set of values for this field, it would make more sense for this to be a dropdown or radio buttons instead of a text field.